routes/libraries.js

```diff
@@ -2,6 +2,25 @@ import { Router } from "express";
 const router = Router();
 import { libraryData } from "../data/index.js";
 import validation from "../public/js/validators/validation.js";
+import multer from "multer";
+
+const storage = multer.diskStorage({
+  destination: (req, file, cb) => cb(null, 'public/uploads/'),
+  filename: (req, file, cb) => {
+      let extension = file.originalname.split('.')[1];
+      console.log(`${extension} - is the extension`)
+      // if(extension!= "jpeg" && extension!= "jpg" && extension!= "png" && extension!= "pdf"){
+      //   throw `VError: photo input must have the extention .jpeg, .jpg, .png or .pdf`
+      // }
+      if (!extension) extension = "";
+      else extension = "." + extension;
+      console.log("INSIDE MULTER STORAGE")
```

Comment:
I think you don't need this anymore, u can import from the image router where you have exported the upload variable